.appveyor.yml

```diff
@@ -1,41 +1,43 @@
 init:
- # -ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
- 
+# -ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
+
+version: 1.0.0.{build}
+
 configuration:
   - Release
   - Debug
 
 image:
   - Visual Studio 2015
-  - Visual Studio 2013
 
 platform:
   - x64
+  - x86
 
 build_script:
-  # build pthread
-  - git clone https://github.com/GerHobbelt/pthread-win32.git C:/pthread-src
-  - ps: "if ( $Env:APPVEYOR_BUILD_WORKER_IMAGE -eq 'Visual Studio 2013' ) { $VS_VERSION='2013' } else { $VS_VERSION='2015' }"
-  - ps: "msbuild C:/pthread-src/pthread.${VS_VERSION}.sln /p:Configuration=$Env:CONFIGURATION /p:Platform=x64"
-  - ps: mkdir C:/pthread-win32
-  - ps: mkdir C:/pthread-win32/include
-  - ps: mkdir C:/pthread-win32/lib
-  - ps: cp C:/pthread-src/*.h C:/pthread-win32/include
-  - ps: cp C:/pthread-src/bin/x64_MSVC${VS_VERSION}.${Env:CONFIGURATION}/pthread_lib.lib C:/pthread-win32/lib
+  # prep shared variables
+  - ps: if ( $Env:PLATFORM -eq 'x86' ) { $DEVENV_PLATFORM = 'Win32' } else { $DEVENV_PLATFORM = 'x64' }
+  - ps: if ( $Env:PLATFORM -eq 'x86' ) { $FOLDER_PLATFORM = 'win32' } else { $FOLDER_PLATFORM = 'win64' }
+  - ps: if ( $Env:APPVEYOR_BUILD_WORKER_IMAGE -eq 'Visual Studio 2013' ) { $VS_VERSION='2013' } else { $VS_VERSION='2015' }
```

Comment:
Visual Studio 2013 is deleted.
`if ( $Env:APPVEYOR_BUILD_WORKER_IMAGE -eq 'Visual Studio 2013' )`